R/delta.R

```diff
@@ -1,14 +1,198 @@
 
 
-#' Title
+#' Create a delta data.frame template
 #'
 #' @description
-#' TODO
+#' Creates a data.frame in the format required by [analyse()] for the use
+#' of applying delta to perform a tipping point analysis.
 #'
-#' @param imputations TODO
+#' @details
+#' To perform a tipping point analysis the [analyse()] function expects
+#' a delta data.frame with 3 variables; `vars$subjid`, `vars$visit` and `delta`
+#' (where `vars` is the object supplied in the original call to [draws()]
+#' as created by the [set_vars()] function).
 #'
+#' These functions will return a data.frame with the aforementioned variables with 1
+#' row per subject per visit. If the `delta` argument to this function is null
```

Comment:
Sorry, should "null" be `NULL`? Why we don't use the coding format "``" for words like NULL, data.frame etc..?

Reply:
fixed